synedrion/src/paillier/encryption.rs

```diff
@@ -228,9 +230,10 @@ impl<P: PaillierParams> CiphertextMod<P> {
         // where `m` is the plaintext, `C` is the ciphertext,
         // `N` is the Paillier composite modulus,
         // `phi` is the Euler totient of `N`, and `mu = phi^(-1) mod N`.
+        // Note that ϕ(N) is always invertible, because `gcd(ϕ(N), N) = 1` by construction.
 
         // `C^phi mod N^2` may be 0 if `C == N`, which is very unlikely for large `N`.
-        // Note that `C^phi mod N^2 / N < N`, so we can unwrap when converting to `Uint`
+        // Note that `C^phi mod N^2 / N < N`, so we can unwrap when converting to `Uint` <–– (DP) This assumes that `gcd(C, N) = 1`, but why is this guaranteed?
```

Comment:
Not sure what you mean here, if `y = x mod N^2` then `y < N^2`, which means `y / N < N`. Hhow does GCD factor in?

Reply:
I honestly no longer remember why I wrote that. Pff. :/